Core/Core/API/APIConversation.swift

```diff
@@ -150,3 +150,109 @@ extension APIConversationMessage {
     }
 }
 #endif
+
+public struct GetConversationsUnreadCountRequest: APIRequestable {
+    public struct Response: Codable {
+        public let unread_count: UInt
```

Comment:
`Int` does not work?  I'm not understanding the why?

Reply:
`Int` includes negative numbers. It doesn't make sense for there to ever be negative unread count, so `UInt` is a stricter representation of it that doesn't allow for invalid negative numbers. It's the same why we use `enum`s instead of `String`s. I could leave it a `String` as it comes back from the api, but then it can possibly be invalid in lots more ways.